src/components/form-textarea/form-textarea.js

```diff
@@ -92,43 +92,42 @@ export default {
       const el = this.$el
 
       // Element must be visible (not hidden) and in document
-      // *Must* be checked after above checks
+      // Must be checked after above checks
       if (!isVisible(el)) {
         return null
       }
 
-      // Remember old height (includes `px` units) and reset it temporarily to `auto`
-      const oldHeight = el.style.height
-      el.style.height = 'auto'
```

Comment:
Part of the reason for using `auto` was that it would trigger a reflow of the `<textarea>`.